include/fmt/std.h

```diff
@@ -59,7 +59,7 @@
 #  endif
 #endif
 
-#ifdef __cpp_lib_filesystem
+#if !defined(FMT_DISABLE_STD_FILESYSTEM) && defined(__cpp_lib_filesystem)
```

Comment:
Let's follow the {fmt} convention and replace `FMT_DISABLE_STD_FILESYSTEM` with `FMT_CPP_LIB_FILESYSTEM` that is initialized to `__cpp_lib_filesystem` if the latter is defined and `FMT_CPP_LIB_FILESYSTEM` is not defined by the user:

```c++
#ifndef FMT_CPP_LIB_FILESYSTEM
# ifdef __cpp_lib_filesystem
#   define FMT_CPP_LIB_FILESYSTEM __cpp_lib_filesystem
# else
#   define FMT_CPP_LIB_FILESYSTEM 0
# endif
#endif

Then you can disable filesystem via `-DFMT_CPP_LIB_FILESYSTEM=0`.

And similarly with `__cpp_lib_variant`.

Reply:
Thank you for the review, I've made the correction.